demos/composition/cyclow/es6/src/confirm.js

```diff
@@ -0,0 +1,34 @@
+import {Block} from 'cyclow'
+
+const viewConfirm = () => [
+  {tag: 'button', content: 'Confirm', on: {click: 'confirm'}},
+  {tag: 'button', content: 'Cancel', on: {click: 'cancel'}}
+]
+
+const viewDefault = mode => ({
+  tag: 'button',
+  content: 'Submit',
+  attrs: {disabled: mode === 'disabled'},
+  on: {click: 'maybe'}
+})
+
+const Confirm = () => Block({
+  inputs: ['disabled'],
+  outputs: ['confirmation'],
+  on: {
+    init: () => state => ({mode: 'disabled'}),
+    disabled: (disabled, {mode}) => {
```

Comment:
Is the signature of this function `(inputData, currentState)` because `disabled` is listed in `inputs`?